server/athenian/api/controllers/jira.py

```diff
@@ -100,12 +100,10 @@ async def get_jira_installation(account: int,
                       .where(and_(JIRAProjectSetting.account_id == account,
                                   JIRAProjectSetting.enabled.is_(False)))),
         mdb.fetch_all(
-            select([Issue.project_id, Issue.type])
-            .select_from(join(Epic, Issue, and_(Epic.acc_id == jira_id,
-                                                Issue.acc_id == jira_id,
-                                                Epic.id == Issue.id),
-                              isouter=True))
-            .distinct()),
+            select([Issue.project_id, Issue.type]).select_from(
+                join(Epic, Issue, Epic.id == Issue.id, isouter=True),
+            ).where(and_(Epic.acc_id == jira_id, Issue.acc_id == jira_id)).distinct(),
```

Comment:
This won't work, unfortunately, because the primary keys in both tables start with `acc_id`.
Return my old code with the only change that `Epic.acc_id == jira_id` appears in `where`:
```suggestion
            select([Issue.project_id, Issue.type])
            .select_from(join(Epic, Issue, and_(Epic.acc_id == Issue.acc_id,
                                                Epic.id == Issue.id),
                              isouter=True))
            .where(Epic.acc_id == jira_id)
            .distinct(),
```